src/definitions.jl

```diff
@@ -561,3 +576,58 @@ Pre-plan an optimized real-input unnormalized transform, similar to
 the same as for [`brfft`](@ref).
 """
 plan_brfft
+
+##############################################################################
+
+region(p::Plan) = p.region
+region(p::ScaledPlan) = region(p.p)
+
+# Projection style (:none, :real, or :real_inv) to handle real FFTs
+function projection_style end
```

Comment:
See above, is this really needed? Maybe it would be better to just implement `output_size` and `*(::AdjointPlan, ::AbstractArray)` for specific sub-types. This would also ensure type stability.